maturity-level-one/README.md

```diff
@@ -150,20 +150,28 @@ Every API should have documentation in the OpenAPI format. If you want to genera
 
 Here is an example on how to generate them with Swashbuckle
 ```csharp
-/// <summary>
-///     Get Health
-/// </summary>
-/// <remarks>Gets the current health status of the API</remarks>
-[HttpGet]
-[Route("health")]
-[SwaggerOperation("Health_Get")]
-[SwaggerResponse(HttpStatusCode.OK, "API is up & running")]
-[SwaggerResponse(HttpStatusCode.InternalServerError, "API is not available")]
-public IHttpActionResult Get()
-{
-    return Ok();
-}
+        /// <summary>
+        /// Get the profiles of the players
+        /// </summary>
+        /// <param name="topPlayersOnly">Indicates whether to return the top players only</param>
+        /// <remarks>Operation description here</remarks>
+        /// <returns>Return a list of Players</returns>
+        [HttpGet(Name = "Players_GetPlayers")]
+        [SwaggerResponse((int)HttpStatusCode.OK, "List of players")]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, "API is no
```

Comment:
I would re-add an example of the body and where the parameters come in such as:
```csharp
public IHttpActionResult GetPlayers(bool topPlayersOnly)	
{	
    return Ok();	
}
´´´

This indicates that OperationId is composed of `{Controller}_{OperationName}`

Reply:
ok, agree